src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -60,17 +75,55 @@ object ScienceDatasets {
     omnibusMtfGradeEightDevQuestionsWithBuscBackground
   ), "/efs/data/dlfa/processed/omnibus_4_8_dev/")
 
+  /////////////////////////////////////////////////////////////////////
+  // Diagram Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val diagramMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.ai2diagramSentences_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10)
   )
 
+  /////////////////////////////////////////////////////////////////////
+  // OpenQA Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val openQaMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.openQaAnimalAndGeneralScience_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10))
 
+  /////////////////////////////////////////////////////////////////////
+  // Omnibus QA
+  /////////////////////////////////////////////////////////////////////
+
+  /*
+   * Fourth Grade BUSC Background Files
+   */
+
+  val omnibusQaGradeFourTrainBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourTrainSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  val omnibusQaGradeFourDevBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourDevSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  val omnibusQaGradeFourTestBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourTestSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  /*
+   * Fourth Grade Questions with BUSC Background (no multiple choice options)
```

Comment:
@matt-gardner I never quite figured out what these classes do (and correspondingly the `Eighth Grade Questions with BUSC Background` section), would you happen to know?